.github/workflows/nightly-link-checker.yml

```
@@ -0,0 +1,53 @@
+name: Nightly Job
+
+# on:
+#  schedule:
+#    - cron: "1 5 * * *" # This schedule runs every night at 12:01 AM Eastern time (adjust according to your timezone)
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: '--max_old_space_size=3584'
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout repository!
+        uses: actions/checkout@v3
+      - name: Setup node - ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+         node-version: ${{ matrix.node-version }}
+         cache: 'npm'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run script and capture logs  #broken link checker script
+        run: yarn build
+
+      - name: Run link checker #broken link checker script
+        run: yarn broken-link-checker:external > output.txt 2>&1 || true
+
+      - name: Extract URLs from log
+        run: |
+          cat output.txt | grep -Eo 'https?://[^ ]+' > urls.txt
+        continue-on-error: true
+
+      - name: Read URLs from file
+        id: read-urls
+        run: echo "::set-output name=urls::$(cat urls.txt)"
```

GH depreciates `::set-output` soon, let's print the contents of `urls.txt` file
```suggestion
        run: cat urls.txt
```

---

update it to
`run: cat filtered_urls.txt`